src/Sylius/Bundle/CoreBundle/Doctrine/ORM/OrderRepository.php

```diff
@@ -471,4 +472,56 @@ private function createQueryBuilderBetweenDates(\DateTime $from, \DateTime $to,
 
         return $queryBuilder;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function findOneByChannelAndCustomer(ChannelInterface $channel, CustomerInterface $customer)
```

Comment:
Actually, this method should be removed, as I dropped the existing cart provider for now. The Cart / Order mess will be fixed in another PR.